database/factories/SkillDeclarationFactory.php

```diff
@@ -15,6 +15,6 @@
             return factory(Applicant::class)->create()->id;
         },
         'skillable_type' => 'applicant',
-        'description' => $faker->text(100),
+        'description' => $faker->words(50, true),
```

Comment:
This will actually be prone to occasional failure. `text()` has a hard limit, while `words()` will generate 50 words of a varying number of characters.

It was set to `$faker->paragraphs(3, true),` before, but I swapped it to `text()`

[More deets here](https://github.com/fzaninotto/Faker#fakerproviderlorem)

Reply:
That's what we want though right, since we're using the WordLimitRule at max 100 words

Reply:
Ah okay, I thought it was a 100 character limit. Either way though, 100 characters won't fail a 100 word count validation check, so I don't think how it was would be prone to random failures.

Reply:
Ya, that's true. Well hopefully removing the Strategic Response Validator will fix it.